client/wfm/src/result/WorkOrderResult.ts

```diff
@@ -0,0 +1,11 @@
+import {StepResult} from './StepResult';
+export interface WorkOrderResult {
+  id: string;
+  status: string;
+  nextStepIndex: number;
```

Comment:
This field sounds like something that we actually should rename/migrate/port. WDYT?
Seen logic for `nextStepIndex` and I do not think that will be a problem:
https://github.com/feedhenry-raincatcher/raincatcher-angularjs/search?utf8=%E2%9C%93&q=nextStepIndex&type=